feature-libs/cart/import-export/core/config/default-import-export-config.ts

```diff
@@ -23,12 +23,16 @@ export const defaultImportExportConfig: ImportExportConfig = {
       ],
       messageEnabled: true,
       downloadDelay: 1000,
-      fileName: 'cart',
+      fileOptions: {
+        fileName: 'cart',
+        extension: 'csv',
+        type: 'text/csv;charset=utf-8;',
+      },
```

Comment:
The key `fileOptions` don't bring much value IMHO. And we should avoid nesting config unless it's justified - just to make it simpler to use for customers (intellisense don't hint on the nested options immediately). 

What about flattening all to:
```
downloadDelay: 1000,
fileName: 'cart',
fileExtension: 'csv',
fileType: 'text/csv;charset=utf-8;',
```
?

Reply:
we decided to separate it as metadata of file because this part is used in method download (independent of delay mechanism)